spec/node/asciidoctor.spec.js

```diff
@@ -111,6 +111,15 @@ describe('Node.js', function () {
       var abstractSectionBlocks = doc.findBy({'context': 'section'}, function (b) { return b.getTitle() === 'Second Section'; });
       expect(abstractSectionBlocks.length).toBe(1);
     });
+
+    it('should be able to find blocks with line number', function () {
```

Comment:
Here I want to make sure that not all blocks have a line number defined.
I've added two more tests to assert that:

*  line number is defined when sourcemap is enabled 
* line number is undefined when sourcemap is disabled